features/deactivation/impl/src/main/kotlin/io/element/android/features/logout/impl/AccountDeactivationView.kt

```diff
@@ -0,0 +1,324 @@
+        ) {
+            Content(
+                state = state,
+                onSubmitClick = {
+                    eventSink(AccountDeactivationEvents.DeactivateAccount(isRetry = false))
+                }
+            )
+            Spacer(modifier = Modifier.height(32.dp))
+            Buttons(
+                state = state,
+                onSubmitClick = {
+                    eventSink(AccountDeactivationEvents.DeactivateAccount(isRetry = false))
+                }
+            )
+        }
+    }
+    AccountDeactivationActionDialog(
+        state.accountDeactivationAction,
+        onConfirmClick = {
+            eventSink(AccountDeactivationEvents.DeactivateAccount(isRetry = false))
+        },
+        onRetryClick = {
+            eventSink(AccountDeactivationEvents.DeactivateAccount(isRetry = true))
+        },
+        onDismissDialog = {
+            eventSink(AccountDeactivationEvents.CloseDialogs)
+        },
+    )
+}
+
+@Composable
+private fun ColumnScope.Buttons(
+    state: AccountDeactivationState,
+    onSubmitClick: () -> Unit,
+) {
+    val logoutAction = state.accountDeactivationAction
+    Button(
+        text = stringResource(CommonStrings.action_deactivate),
+        showProgress = logoutAction is AsyncAction.Loading,
+        destructive = true,
+        enabled = state.submitEnabled,
+        modifier = Modifier.fillMaxWidth(),
+        onClick = onSubmitClick,
+    )
+}
+
+@Composable
+private fun Content(
+    state: AccountDeactivationState,
+    onSubmitClick: () -> Unit,
+) {
+    val isLoading by remember(state.deactivateFormState) {
+        derivedStateOf {
+            state.accountDeactivationAction is AsyncAction.Loading
+        }
+    }
+    val eraseData = state.deactivateFormState.eraseData
+    var passwordFieldState by textFieldState(stateValue = state.deactivateFormState.password)
+
+    val focusManager = LocalFocusManager.current
+    val eventSink = state.eventSink
+
+    Column(
+        modifier = Modifier.fillMaxWidth(),
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+    ) {
+        Text(
+            text = stringResource(R.string.screen_deactivate_account_description),
+            style = ElementTheme.typography.fontBodyMdRegular,
+            color = ElementTheme.colors.textSecondary,
+        )
+        InfoListOrganism(
+            items = persistentListOf(
+                InfoListItem(
+                    message = stringResource(R.string.screen_deactivate_account_list_item_1),
+                    iconComposable = {
+                        Icon(
+                            modifier = Modifier.size(20.dp),
+                            imageVector = CompoundIcons.Close(),
+                            contentDescription = null,
+                            tint = ElementTheme.colors.iconCriticalPrimary,
+                        )
+                    },
+                    iconVector = CompoundIcons.Close(),
```

Comment:
This is the cause of the tint issue :). Will remove in a coming commit.

Reply:
Oh, I didn't notice the green X 😅 :

<details>
<summary>Details</summary>

<img width="461" alt="image" src="https://github.com/user-attachments/assets/37d9f9b8-71cb-4c85-aeab-6a0af6d04d78">


</details>